api/src/pcapi/admin/custom_views/venue_view.py

```diff
@@ -45,6 +50,26 @@ class VenueChangeForm(Form):
     )
 
 
+class VenueCriteriaFilter(fa_filters.BaseSQLAFilter):
+    """
+    Filter venues based on tag (criterion) name.
+    """
+
+    def apply(self, query: Query, value: str, alias=None) -> Query:
+        parsed_value = tools.parse_like_term(value)
+        venue_ids = [
+            vc.venueId
+            for vc in offerers_models.VenueCriterion.query.join(Criterion).filter(Criterion.name.ilike(parsed_value))
+        ]
+        return query.filter(self.column.in_(venue_ids))
```

Comment:
Ca marche bien en staging avec plein de venues liées à des criterion ? Je me demande si on ne charge pas des masses de `venueId` pour ensuite filtrer dedans.
Est-ce qu'il n'y a pas moyen de faire plutôt un truc genre:
```suggestion
        return query.join(offerers_models.VenueCriterion).join(Criterion).filter(Criterion.name.ilike(parsed_value))
```